site/pages/reference/account-kit/infra/functions/alchemyGasManagerMiddleware.mdx

```diff
@@ -18,13 +18,13 @@ import { alchemyGasManagerMiddleware } from "@account-kit/infra";
 ## Usage
 
 ```ts
-import { sepolia, alchemyGasManagerMiddleware } from "@account-kit/infra";
+import { sepolia, alchemyErc7677Middleware } from "@account-kit/infra";
 import { http } from "viem";
 
 const client = createSmartAccountClient({
   transport: http("rpc-url"),
   chain: sepolia,
-  ...alchemyGasManagerMiddleware("policyId"),
```

Comment:
oh I saw this in a another PR somewhere too... This looks like a bug in main 🤔

Reply:
I think this is a git issue somewhere - it was fixed in a different PR that's already merged